src/RuleSet/Loader/Composer/Package.php

```diff
@@ -42,7 +42,7 @@ public static function fromArray(array $package)
      */
     public function generatePackageKey()
     {
-        return str_replace('/', '_', $this->name).'_'.$this->version;
+        return str_replace(array('/', '\\', ':'), '_', $this->name).'_'.$this->version;
```

Comment:
I think this looks all right, I'm used to having full paths converted with 2 underscores after drive letter on windows systems.

![image](https://cloud.githubusercontent.com/assets/6818055/23675585/d08ba624-0347-11e7-9a17-428c08081b02.png)